### DISCUSSION TITLE
Change drive processing

### DISCUSSION THREAD

**Discussion by SirMatrixNico** [lllyasviel/Fooocus#2274] [Q&A] [answered]
What a good day everyone,

I would like you to help me change the processing of the photographs, which is what I need, I explain. I have 2 disk drives, one SSD and one NV2. I have the Fooocus software on the NV2 because it is faster, but when I process the photographs the SSD disk is the one used for processing and it is slower. How can I change to the NV2? Thanks for your help.
![SSD](https://github.com/lllyasviel/Fooocus/assets/159728157/a56bb463-d99c-4fdc-83df-205fcfbb5d9a)

**Comment by mashb1t** [comment, marked as answer]
Make sure your swap is located on the NVMe (M.2) by changing it in the system settings. You can follow the instructions in the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap) and then not use automatic, but manual swap placement.

**Reply by SirMatrixNico** [reply]
Thanks, I already see more activity in NV2 with the parameters you mentioned. I appreciate the help.
